content/docs/aws/iam-for-sa.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Authenticating Kubeflow to AWS"
```

Comment:
A general comment about this page: As a reader, I'm not sure what I need to do and what is background information. Can you split the page into the steps I should take (with examples), and a section for background information?

Reply:
Got you. Thanks sharing feedbacks from users without background on this. I may jump to technical side directly. Let me add some more background details